### PR TITLE
Fix dashboard empty state issues

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- Our default dashboard was broken because no insight was set for most of them, which meant that those items would show up as below. Considered a migration but that would probably lock the table, so went for an in-api solution which should be fine
 
image

- Remove 'Weekly revenue' as that would always be empty anyway as we don't have a revenue event
- When searching dashboards show the empty table rather than 'create your first dashboard'
- Update styling of 'create your first dashboard'

## How did you test this code?

- Manual testing for the search (go to dashboard, type in `asdfasdfadsf`, see that it's an empty table)
- Manual testing for the dashboard (remove all dashboards, see 'create your first dashboard', click app dashboard, verify the items work)
- Unit tests

### CODE REVIEW

**Comment by macobo** [comment]
> Considered a migration but that would probably lock the table

Note there's a way to do these safely, basically commit row-by-row, but we might instead run into timeouts instead because of that.

**Comment by timgl** [comment]
> Note there's a way to do these safely, basically commit row-by-row, but we might instead run into timeouts instead because of that.

Yeah, I figured it wasn't worth the risk and this was a decent solution.

File: frontend/src/scenes/feature-flags/FeatureFlag.tsx [hunk 1]
```diff
@@ -158,6 +158,9 @@ export function FeatureFlag(): JSX.Element {
                         if (featureFlagId !== 'new' && newValues.key) {
                             setHasKeyChanged(newValues.key !== featureFlag.key)
                         }
+                        if (newValues.key) {
```
Comment (macobo) [inline frontend/src/scenes/feature-flags/FeatureFlag.tsx:161]: Q: Unrelated fix?

**Review by macobo — APPROVED** [review]
Other than the seemingly unrelated ff fix, lgtm.

File: frontend/src/scenes/feature-flags/FeatureFlag.tsx (hunk 1 above)
Comment (timgl) [reply frontend/src/scenes/feature-flags/FeatureFlag.tsx]: Oops, that wasn't meant to be committed
